ui/src/app/components/help-sidebar.tsx

```diff
@@ -381,14 +394,30 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile(), wi
 
     renderCriteria() {
```

Comment:
I went into a little more detail offline, but I think we should use the existing structure to display this content.